testing/py3-lib_users/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname=py3-lib_users
+_pkgname=lib_users
+pkgver=0.13
+pkgrel=0
+pkgdesc="Checks /proc for libraries being mapped but marked as deleted"
+url="https://github.com/klausman/lib_users"
+arch="noarch"
+license="GPL-2.0-or-later"
+depends="python3"
+makedepends="py3-setuptools"
+source="$pkgname-$pkgver.tar.gz::https://github.com/klausman/lib_users/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	python3 setup.py build
+}
+
+package() {
```

Comment:
this should be just the python3 setup.py install command from the _py() function